[{"name": "test.test_reverse_dictionary.ReverseDictionary.test_first", "status": "errored", "message": "too many values to unpack (expected 2)", "passed": false, "points": ["p01-13.1"], "backtrace": ["  File \"/usr/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"/usr/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n", "  File \"/home/du_ds/Documents/Git/PythonDataAnalysisCourse/hy-data-analysis-with-python-summer-2019/part01-e13_reverse_dictionary/test/test_reverse_dictionary.py\", line 20, in test_first\n    r=reverse_dictionary(d)\n", "  File \"/home/du_ds/Documents/Git/PythonDataAnalysisCourse/hy-data-analysis-with-python-summer-2019/part01-e13_reverse_dictionary/src/reverse_dictionary.py\", line 4, in reverse_dictionary\n    for key, value in d:\n"]}, {"name": "test.test_reverse_dictionary.ReverseDictionary.test_second", "status": "errored", "message": "too many values to unpack (expected 2)", "passed": false, "points": ["p01-13.1"], "backtrace": ["  File \"/usr/lib/python3.6/unittest/case.py\", line 59, in testPartExecutor\n    yield\n", "  File \"/usr/lib/python3.6/unittest/case.py\", line 605, in run\n    testMethod()\n", "  File \"/home/du_ds/Documents/Git/PythonDataAnalysisCourse/hy-data-analysis-with-python-summer-2019/part01-e13_reverse_dictionary/test/test_reverse_dictionary.py\", line 29, in test_second\n    r=reverse_dictionary(d)\n", "  File \"/home/du_ds/Documents/Git/PythonDataAnalysisCourse/hy-data-analysis-with-python-summer-2019/part01-e13_reverse_dictionary/src/reverse_dictionary.py\", line 4, in reverse_dictionary\n    for key, value in d:\n"]}]